soccer/gameplay/tactics/pass_to_point_heuristic.py

```diff
@@ -0,0 +1,7 @@
+import constants
+import robocup
+import time
+
+
+def pass_to_point_heuristic(x: int, y: int) -> robocup.Point:
```

Comment:
I don't understand this.... what are `x` and `y` supposed to be?

Reply:
They are x and y coordinates on the field. See the comment I left for Joe above for reasoning.